includes/specials/SpecialBlockList.php

```diff
@@ -244,7 +243,6 @@ function getFieldNames() {
 	}
 
 	function formatValue( $name, $value ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this function "formatValue" is 21 which is greater than 20 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1541)